sdk/core/core/inc/az_http_query.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
General note:

These headers are really small. This is fine, but can create extra work down the line. Will anyone actually want to use just some of these headers?